Add receipt for pif package

oliverepper opened the pull request:
### Brief summary of what the package does

pif.el prevents the initial “Flash of Light” during Emacs startup.

In addition to reducing the “Flash of Light”, it ensures that the initial frame appears in the correct size and position – just as it was when Emacs was closed. Furthermore, it hides GUI elements such as the modeline, fringes, and the cursor during startup for a distraction-free initialization.

### Direct link to the package repository

https://github.com/oliverepper/pif

### Your association with the package

I am the author

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

oliverepper commented:
I ran melpazoid against the PR and got this:

```pre
1 issue found:
11:3: warning: You should include standard keywords: see the variable `finder-known-keywords'.
```

I saw that before while running the linters. So I decided on these keywords on purpose. I saw non-standard keywords in other packages and figured that it might help discoverability. I can change that back to use just the keyword "convenience" if that is more appropriate?

riscy commented:
I think the gist here is that *if* you're going to add `Keywords`, you should try to add at least *one* keyword as listed in the variable `finder-known-keywords`.  You can add additional novel keywords of your own, but one must be from that controlled list.

So go ahead and use your own keywords but including `convenience` and maybe even `faces` ("fonts and colors for text") would solve that linter issue for you.

Let me double check this: pif.el#L3: Have you done the paperwork to assign this copyright?

And lastly, how do users use this?  I feel like that detail is missing from your `;;; Commentary`.

oliverepper commented:
I have updated the keywords. Thanks for the explanation.

Regarding pif.el#3 that was just a copy&paste error. I would agree to assign the copyright to the GNU but have not done so. I have updated the line.

I added an explanation and a sample config to the `;;; Commentary` section.

Thanks a lot for your time and comments!

riscy commented:
Thanks!  In case you haven't, ensure GitHub release 0.0.5 is up-to-date with your current code and `Package-Version`.  I'll merge this now.
